playwright/README.md

```diff
@@ -0,0 +1,251 @@
+# Playwright E2E Tests
+
+## Installation
+
+### Prerequisites
+
+Ensure you have Node.js (>= 18.12.0) and Yarn (>= 1.22.19) installed on your system.
+
+### Install Dependencies
+
+From the project root directory, install all dependencies including Playwright:
+
+```bash
+# Install all project dependencies (including Playwright)
+yarn install
+
+# Install Playwright browsers
+yarn playwright install
+
+# Optional: Install only Chromium browser for faster setup
+yarn playwright install chromium
+```
+
+### Verify Installation
+
+You can verify that Playwright is properly installed by running:
+
+```bash
+# Check Playwright version
+yarn playwright --version
+
+# List available tests (without running them)
+yarn playwright test --list
+```
+
+## Setup
+
+### Environment Variables
+
+The tests support multiple authentication methods and cloud provider configurations. All environment variables should be configured in the `playwright.env.json` file.
+
+#### Authentication Methods
+
+The tests support:
+- **Username/Password Authentication**: Standard Red Hat SSO login
+
+1. **Username/Password Authentication (Primary)**:
+   - `TEST_WITHQUOTA_USER`  - Username for test authentication
+   - `TEST_WITHQUOTA_PASSWORD`  - Password for test authentication
+
+#### Configuration Files
+
+The test configuration uses `playwright.env.json` for environment-specific settings. All authentication credentials, cloud provider settings, infrastructure configurations, and test environment options are mapped in this file.
+
+#### Example `playwright.env.json` Structure
+
+```json
+{
+  "TEST_WITHQUOTA_USER": "your-username@example.com",
+  "TEST_WITHQUOTA_PASSWORD": "your-password",
+  "GOV_CLOUD": "false",
+  "BROWSER": "firefox",
+  "BASE_URL": "https://console.dev.redhat.com/openshift/"
+  .....
+}
+```
+
+**Note**: Create your own `playwright.env.json` file based on this structure. This file is not part of the repository and should not be committed to version control as it contains sensitive credentials.
+
+
+
+
+### Running Tests
+
+```bash
+# Run all tests
+yarn playwright test
+
+# Run specific test file
+yarn playwright test playwright/e2e/clusters/register-cluster.spec.ts
+
+# Run tests for specific directory
+yarn playwright test playwright/e2e/downloads/
+
+# Run with specific browser
+BROWSER=chromium yarn playwright test
+
+# Run with UI mode (interactive)
+yarn playwright test --ui
+
+# Run in headed mode (see browser)
+yarn playwright test --headed
```

Comment:
you added specific scripts in package.json for playwright. we should document how to use them here instead of calling playwright directly, otherwise they are useless.